ugwumadu116/Auto-Mart-Andela
# #166115507 build out user can view available cars with in a specific price range

#### What does this PR do?
- Creates a route to get all available cars within a specific price range
#### Description of Task to be completed?
- Create a route that uses GET /car/ eg `GET /api/v1/car?status=available&min_price=100&max_price=1000   
- Create a car service to handle all car functionalities
- Create a car controller to handle route request
- Create a user validator to know the user that is making the request
#### How should this be manually tested?
- Sign in with firstName, lastName, email, and password
   - exmaple `{
	"firstName": "joel",
	"lastName": "ugwumadu2",
	"email": "ugwumadu116@gmail.com",
	"password": "test"
	"address": "ipaja road"

}`
- Login in with your email and password 
  - example `{
	"email": "ugwumadu116@gmail.com",
	"password": "test"
}`
- Use your login token and set it in Headers and set the key to be authorization while the value will be your token
- Then make a GET request to `/api/v1/car?status=available&min_price=100&max_price=1000    `
- Your messages will appear in your response
#### What are the relevant pivotal tracker stories?
#166115507
